### DISCUSSION TITLE
WARNING: MYSQL_OPT_RECONNECT is deprecated and will be removed in a future version.

### DISCUSSION THREAD

**Discussion by jphz** [pocoproject/poco#4272] [Q&A]
Where is this warning closed and are there any alternative solutions in the future?

Server Info: 8.0.35
Server Version: 80035
Host Info: 127.0.0.1 via TCP/IP

**Comment by edison12a** [comment]
Same Issue: 
How can I get rid of this warning?

**Comment by edison12a** [comment]
ping!
